### ISSUE TITLE
graphql-code-generator doesn't work with meteor-graphql-rxjs

### ISSUE THREAD

**Issue by darkbasic** [dotansimha/graphql-code-generator#159]
This is with the `http://localhost:3000/graphql` endpoint, using `apollo-server-express`:
```
[niko@arch-desktop angularcli-graphql]$ npm run generator

> angularcli-graphql@0.0.0 generator /home/niko/WebstormProjects/angularcli-graphql
> gql-gen --url http://localhost:3000/graphql --template ts --out ./src/types.d.ts "./src/graphql/**/*.ts"

Loading GraphQL Introspection from remote: http://localhost:3000/graphql...
Generated file written to /home/niko/WebstormProjects/angularcli-graphql/src/types.d.ts
```

This is with the `http://localhost:3000/graphql_live` endpoint, using `meteor-graphql-rxjs`:
```
[niko@arch-desktop angularcli-graphql]$ npm run generator

> angularcli-graphql@0.0.0 generator /home/niko/WebstormProjects/angularcli-graphql
> gql-gen --url http://localhost:3000/graphql_live --template ts --out ./src/types.d.ts "./src/graphql/**/*.ts"

Loading GraphQL Introspection from remote: http://localhost:3000/graphql_live...
/home/niko/WebstormProjects/angularcli-graphql/node_modules/graphql-code-generator/dist/loaders/introspection-from-url.js:43
                reject('Unable to download schema from remote: ' + body.errors.map(function (item) { return item.message; }).join(', '));
                                                                               ^

TypeError: Cannot read property 'map' of undefined
    at Request._callback (/home/niko/WebstormProjects/angularcli-graphql/node_modules/graphql-code-generator/dist/loaders/introspection-from-url.js:43:80)
    at Request.self.callback (/home/niko/WebstormProjects/angularcli-graphql/node_modules/request/request.js:188:22)
    at emitTwo (events.js:125:13)
    at Request.emit (events.js:213:7)
    at Request.<anonymous> (/home/niko/WebstormProjects/angularcli-graphql/node_modules/request/request.js:1171:10)
    at emitOne (events.js:115:13)
    at Request.emit (events.js:210:7)
    at IncomingMessage.<anonymous> (/home/niko/WebstormProjects/angularcli-graphql/node_modules/request/request.js:1091:12)
    at Object.onceWrapper (events.js:312:30)
    at emitNone (events.js:110:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angularcli-graphql@0.0.0 generator: `gql-gen --url http://localhost:3000/graphql_live --template ts --out ./src/types.d.ts "./src/graphql/**/*.ts"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the angularcli-graphql@0.0.0 generator script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/niko/.npm/_logs/2017-10-31T11_50_46_395Z-debug.log
```
To test it, check out the following:
Server: https://github.com/Urigo/WhatsApp-Server-GraphQL-Live-RxJS-Meteor
Run the server with `npm start`
Client: https://github.com/darkbasic/angularcli-graphql
Run the generator with `npm run generator` (edit the url to `http://localhost:3000/graphql_live` first).
Run the client with `ng serve`.

**Comment by dotansimha** [comment]
@darkbasic, GraphiQL is available for the `graphql_live` endpoint? If so, do you see there the GraphQL documentation?

**Comment by darkbasic** [comment]
> GraphiQL is available for the graphql_live endpoint? If so, do you see there the GraphQL documentation?

Yes and yes.

**Comment by dotansimha** [comment]
@darkbasic I will look into it soon. 
I guess that the issue is with the HTTP request, and the fact that Meteor manages the HTTP server (with `WebApp` package).

**Comment by daverickdunn** [comment]
I'm getting the **exact** same error.

I'm using Graphene however, and GraphiQL is running fine. In fact, I have the gql-gen command saved as a script in my package.json, I used it a few times last October with no issues. Since then I've changed nothing about the the command or the server's config. The only thing I've done is add a few new queries/mutations.

I went back through the git commits, the package version is the same now as then: "graphql-code-generator": "^0.8.14"

Can't say the same for dependencies tough.
